.github/workflows/cicd.yml

```diff
@@ -65,18 +65,39 @@ jobs:
           image: ${{ env.TEST_TAG }}
           config: test/container-structure-test.yaml
 
+      - name: Get version from package.json
+        id: get_version
+        run: |
+          VERSION=$(node -p "require('./package.json').version")
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
```

Comment:
the version is written to a file somewhere, `version.ts` I think. Might be clearner to take it from there but idk